src/FsToolkit.ErrorHandling.TaskResult/Task.fs

```diff
@@ -33,3 +33,12 @@ module Task =
   }
 
   let ofUnit (t : Task) = task { return! t }
+
+  let catch (x : Task<_>) =
```

Comment:
nit: Missing a `///` comment

Reply:
That one's a bit trickier to describe, how does it sound now?

Reply:
Looks great! Thanks!